app/facades/food_facade.rb

```diff
@@ -0,0 +1,17 @@
+class FoodFacade
+  def self.search_foods(keyword)
+    data = UsdaService.food_search(keyword)
+    # {
+    #   total_hits: data[:totalHits],
+    #   foods: create_foods(data[:foods])
+    # }
+
+    FoodSearchResult.new(keyword, data[:totalHits], create_foods(data[:foods]))
+  end 
```

Comment:
Wrapping everything you need in one object is totally fine, and I like the idea. An OpenStruct would essentially do the same thing. Its purpose is to wrap things into one object without needing to define a unique class. You can take a look [here](https://ruby-doc.org/stdlib-2.5.1/libdoc/ostruct/rdoc/OpenStruct.html) if you want to try it out.